BAZEL.md

```diff
@@ -0,0 +1,116 @@
+# BAZEL usage instructions
+
+To build a C# gapic client library from a proto service description,
+the following files are required:
+
+`WORKSPACE.bazel`:
```

Comment:
Rather than have the content of all these files here in the README, how about having a bazel-example directory that's ready-to-build?

Reply:
Yes, definitely a better idea - I'll abandon this PR and do another...